src/hotspot/cpu/ppc/vm_version_ppc.hpp

```diff
@@ -131,6 +131,8 @@ class VM_Version: public Abstract_VM_Version {
 
   // POWER 8: DSCR current value.
   static uint64_t _dscr_val;
+
+  static void initialize_cpu_information(void);
```

Comment:
This looks wrong.
The original patch has a different change: https://github.com/openjdk/jdk/commit/409a39ec8da83d6a0895e7e213604455ebf50485#diff-0e8cad40dc55410beb083852ab7a431d0c0cd668719b3c252c93c5083b824654R85-R94